MIT Press.js

```diff
@@ -0,0 +1,206 @@
+{
+	"translatorID": "ac277fbe-000c-46da-b145-fbe799d17eda",
+	"label": "MIT Press",
+	"creator": "Guy Aglionby",
+	"target": "https://(www\\.)?mitpress\\.mit\\.edu/",
```

Comment:
again, to distinguish it from journals, make the regex more restrictive (just re-using the bits from detect Web, which you may then be able to simplify)